tsconfig.json

```diff
@@ -3,10 +3,11 @@
     "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
     "outDir": "./lib",                        /* Redirect output structure to the directory. */
-    "rootDir": "./src",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
Could you please explain the reason for this change?
I think this is the cause of the error below.
> Error: Cannot find module '/home/runner/work/checkbox-action/checkbox-action/lib/main.js'. Please verify that the package.json has a valid "main" entry